pkg/mcp/server/server.go

```diff
@@ -201,6 +201,11 @@ func (s *Server) StreamAggregatedResources(stream mcp.AggregatedMeshConfigServic
 	}
 }
 
+// IncrementalAggregatedResources provides the ability to incrementally update the resources on the client.
+func (s *Server) IncrementalAggregatedResources(mcp.AggregatedMeshConfigService_IncrementalAggregatedResourcesServer) error {
+	return nil
```

Comment:
We should return "not implemented" here and in the client_test.go code (see https://github.com/istio/istio/pull/8292/files#diff-79a4c43ff7862192768effb6f814000bR171)